### PR TITLE
Database check returns HTTP 503 on fail. #96

### PR DESCRIPTION
**Pull request by WalterMoar**
Updated the database check to not return 200 on fail. Using a 503 (service unavailable) as a 500 (internal server error) still just doesn't feel right when there is no error internal to the server.

### CODE REVIEW

**Review by bryan-gilbert — APPROVED** [review]
Good to switch to json responses.

File: ppr-api/src/endpoints/healthcheck.py [hunk 1]
```diff
@@ -23,20 +24,23 @@ def health():
...
     Returns a health check for the reachability of the database.
     """
-    return {
-        "status": database_status(session)
-    }
-
-
-def database_status(session: sqlalchemy.orm.Session):
     try:
-        session.execute('SELECT 1')
```
Comment (WalterMoar) [inline ppr-api/src/endpoints/healthcheck.py:37]: For consistency - I'm good with singles or doubles, and prefer doubles, but needs to be consistent.

**Comment by WalterMoar** [comment]
> Good to switch to json responses.

That was Justin - just working from his setup :)

File: ppr-api/src/endpoints/healthcheck.py (hunk 1 above)
Comment (bryan-gilbert) [reply ppr-api/src/endpoints/healthcheck.py:37]: Can the lint tooling fix this? The act of "on save" or "on lint" is the best place to get this consistency.

Comment (WalterMoar) [reply ppr-api/src/endpoints/healthcheck.py:37]: Good question, although you may sometimes want to use single quotes if you have a lot of embedded double quotes. PEP8 doesn't pick a lane on this one, it just recommends consistency.

Comment (jguertin) [reply ppr-api/src/endpoints/healthcheck.py:37]: I'll screw this up over and over again, lol.  Too much switching between languages.  I'm good with double-quotes, except when they exist in the string.  I'll have to see if I can get the formatter to do this. :)

**Review by jguertin — APPROVED** [review]

**Pull request merged**